### PR TITLE
docs: add vocabs in doc

### LINKED ISSUE #235: [docs] Better specify which vocab is supported by which model
Without reading the code, it is not possible currently to know the specific alphabet used, and which model is using which. We need to tackle this

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR adds:
- tab of supported vocabs in doc (dataset)
- vocab to use for each model (models)

Any feedback is welcome!

closes #235

### CODE REVIEW

File: docs/source/models.rst [hunk 1]
```diff
@@ -98,6 +98,8 @@ Identifying strings in images
 All text recognition models above have been evaluated using both the training and evaluation sets of FUNSD and CORD (cf. :ref:`datasets`).
 Explanations about the metrics being used are available in :ref:`metrics`.
 
+All these recognition models are trained with our french vocab (cf. :ref:`vocabs`).
```
Comment (fg-mindee) [inline docs/source/models.rst:101]: If I'm not mistaking, for now, we're using a custom alphabet (not in the same order as the french's one?)

**Review by fg-mindee — APPROVED** [review]
Thanks for the PR! I added a small comment as I believe our pretrained recognition models are still using the former alphabet for now

File: docs/source/models.rst (hunk 1 above)
Comment (charlesmindee) [reply docs/source/models.rst:101]: yes, do I need to change the order in the doc ?

Comment (charlesmindee) [reply docs/source/models.rst:101]: yes, do I need to change the order in the docs ?

Comment (fg-mindee) [reply docs/source/models.rst:101]: No that's alright, we're gonna change the model soon anyway!

**Pull request merged**